hack/build.sh

```
@@ -42,8 +42,8 @@ fi
 # Create the release directory
 mkdir -p "${PROVIDER_ROOT}/release"
 
-for OS in "${PROVIDER_BUILD_PLATFORMS[@]}"; do
-    for ARCH in "${PROVIDER_BUILD_ARCHS[@]}"; do
+for OS in ${PROVIDER_BUILD_PLATFORMS[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)